tranzito_utils_js/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "tranzito_utils_js",
-  "version": "1.0.2",
+  "version": "1.1.5",
```

Comment:
I think, this is the version of `tranzito_utils_js` package not for the gem. We are using this package.json to install the bootstrap which is being used into the gem. 
If we change this version then we also need to publish the JS package as well

Reply:
I think it makes sense for the version of both to be the same. It makes it easier to track what version is the correct version